Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM rust:latest as builder
+
+WORKDIR /shotover-proxy
+
+COPY ./ ./
+
+RUN cargo build --release
+
+FROM debian:bullseye-slim
+
+RUN apt-get update -y \
```

Comment:
isnt this basically a no-op?
updating the lists and then deleting them?
Or is there a reason for it?

Reply:
Oops, there should be an upgrade in there.

Reply:
Is that a normal thing to do?
Wouldnt the debian image already have up to date packages?
Looking at redis dockerfile I can see that it doesnt run upgrade https://github.com/docker-library/redis/blob/master/6.2/Dockerfile

Reply:
https://github.com/docker-library/cassandra/blob/master/4.0/Dockerfile cassandra too

Reply:
Ok sure I'll leave out the upgrading.